lighthouse-cli/bin.ts

```diff
@@ -67,6 +67,7 @@ const cliFlags = yargs
     'mobile',
     'save-assets',
     'save-artifacts',
+    'exclude-artifacts',
```

Comment:
we should try to pick a flag name with a default false value (so that `undefined` or anything falsy will be coerced to it). Maybe just `--include-artifacts`?